### ISSUE TITLE
Feature request: Adding custom message to desktop notification

### ISSUE THREAD

**Issue by bjornasm** [poetaman/arttime#54]
This is possibly linked to https://github.com/poetaman/arttime/issues/36 - but it would be nice to add a custom message to be displayed on the desktop notification. Say if you have a timer to remember to take a walk and drink a glass of water every other hour in between other types of timers it would be nice to have a specific textual reminder for that. I can have a quick look on the source to see if this is something I am able to contribute to.

**Comment by poetaman** [comment]
@bjornasm Yes it is linked to #36. There is a plan for users to be able to specify their timers in more expressive ways, though I don't see it happening anytime soon. There is a workaround one can use for now: arttime's notification does show the name of art. A user can open multiple instances of arttime, and have different reminders in each. If the displayed art in each instances is different, they would be able to get the context of notification.

Check for example the "keypoem" file [share/arttime/keypoems/timer.banana10learn_8b](https://github.com/poetaman/arttime/blob/main/share/arttime/keypoems/timer.banana10learn_8b) below.

```
----
m2 bananas every hour for next 10 hours = 2000 calorie banana diet!
--------
abanana
--------
g1h;loop10
--------
mYou will get 10 notifications, though don't follow this diet :)
```

It can be loaded from command line like this: ``arttime -k timer.banana10learn_8b`` or by loading it from within running arttime instance by pressing ``k`` followed by Tab/arrow-keys based navigation and pressing enter.

The good part is you can create as many such "keypoems" and save them in the same or different directory. The syntax is the same as the keystrokes one would type in arttime's interface to get an effect. Essentially arttime interprets contents of the file as if each key were pressed. ``abanana`` followed by new line changes the art to banana. Art names refer to file names under directory [share/arttime/textart](https://github.com/poetaman/arttime/tree/main/share/arttime/textart). One can easily add a file for "water" or "glass" and write save a "timer.drinkwater_8b" keypoem like this:

```
----
mwater is essential for life, drink it! 
--------
awater
--------
g2h;loop10
```

Supporting info: https://github.com/poetaman/arttime/wiki

**Comment by poetaman** [comment]
@bjornasm Btw... As getting notifications to drink water would be so common, I went ahead and added basic water ASCII art and a keypoem. To give it a try, pull and install the main branch again...  Then either launch arttime to load the program at launch time: ``arttime -k timer.drinkwater_8b`` or after launching arttime press ``k`` followed by Tab/arrow-keys to navigate, and Enter to run the keypoem script.

**Comment by bjornasm** [comment]
Thank you for your extremely thorough reply as well as this added feature @poetaman !
